src/main/java/org/folio/circulation/domain/LoanRepresentation.java

```diff
@@ -52,6 +52,17 @@ public JsonObject extendedLoan(Loan loan) {
     return extendedRepresentation;
   }
 
+  private void addPolicy(JsonObject extendedRepresentation,
+                         CirculationPolicy policy,
+                         String policyName) {
```

Comment:
Please can this use standard line continuation indentation. I think that vertical alignment like this adds maintenance overhead and wastes horizontal space.

Reply:
What is `standard line continuation indentation` ?

Reply:
For FOLIO, at the moment (it is likely to change soon, as we move to the Google Java code standard) it is two-character indentation for line continuation.

Does that make sense?

Reply:
Changed